server/src/main/java/org/elasticsearch/index/engine/LiveVersionMap.java

```diff
@@ -476,11 +476,26 @@ public long ramBytesUsed() {
     }
 
     /**
-     * Returns how much RAM would be freed up by refreshing. This is the RAM usage of the current version map. It doesn't include tombstones
-     * since they don't get cleared on refresh, nor the old version map that is being reclaimed.
+     * Returns how much RAM is used by refresh. This is the RAM usage of the current and old version maps, and the RAM usage of the
+     * archive, if any.
      */
     long ramBytesUsedForRefresh() {
```

Comment:
This name was misleading IMO and since in the current approach we need to distinguish between what is total usage and what is the usage that could be FURTHER reclaimed, I've created reclaimableRamBytes.